src/main/java/com/smartcar/sdk/data/Compatibility.java

```diff
@@ -0,0 +1,37 @@
+package com.smartcar.sdk.data;
+
+/**
+ * A container for the compatibility endpoint
+ */
+public class Compatibility extends ApiData {
+	private boolean compatibility;
```

Comment:
this variable must be named `compatible` otherwise GSON does not find it and set it

currently all responses are pared into `null` and then type cast into falses